CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+    private IJokeService JokeServiceInterface { get;  }
+    private IOutput OutputInterface { get;  }
```

Comment:
I'd probably leave the interface part of the name out. I don't think it's adding anything, plus the underlying type won't be an interface: 
```suggestion
    private IJokeService JokeService { get;  }
    private IOutput Output { get;  }
```